main.tf

```diff
@@ -124,7 +124,7 @@ resource "azurerm_kubernetes_cluster" "main" {
 
 resource "azurerm_log_analytics_workspace" "main" {
   count               = var.enable_log_analytics_workspace ? 1 : 0
-  name                = "${var.prefix}-workspace"
+  name                = var.cluster_analytics_workspace
```

Comment:
`name                =  var.cluster_log_analytics_workspace_name==null?"${var.prefix}-workspace": var.cluster_log_analytics_workspace_name`

Reply:
I applied the suggested changes